uaclient/tests/test_cli_attach.py

```diff
@@ -273,8 +273,9 @@ def fake_request_updated_contract(cfg, contract_token, allow_enable):
             raise error_class(error_str)
 
         request_updated_contract.side_effect = fake_request_updated_contract
-        ret = action_attach(args, cfg=cfg)
-        assert 1 == ret
+        with pytest.raises(SystemExit):
```

Comment:
I think it is useful to check the `code` for the `SystemExit` here is 1, as the old test was doing